contracts/number/types/Fixed18.sol

```diff
@@ -72,6 +72,19 @@ library Fixed18Lib {
         return Fixed18.wrap(Fixed6.unwrap(a) * 1e12);
     }
 
+    /**
+     * @notice Creates a signed fixed-decimal from a significand and an exponent
+     * @param significand Significand
+     * @param exponent Exponent
+     * @return New signed fixed-decimal
+     */
+    function from(Fixed18 significand, int256 exponent) internal pure returns (Fixed18) {
+        if (exponent < 0) {
+            return significand.div(from(int256(10 ** uint256(-1 * exponent))));
+        }
+        return significand.mul(from(int256(10 ** uint256(exponent))));
```

Comment:
super nit: for stuff like this where we're returning a value based on a branch, we usually style it using a ternary like this:
```
return exponent < 0
    ? significand.div(from(int256(10 ** uint256(-1 * exponent))))
    : significand.mul(from(int256(10 ** uint256(exponent))));
```

Reply:
Fixed in [1b08a0a](https://github.com/equilibria-xyz/root/pull/108/commits/1b08a0ae47dff8537a434a3147335697aa800d8f)